demo/list-box-presentation-demos.html

```diff
@@ -27,7 +27,7 @@ <h3>Separators</h3>
     <h3>Customizing listbox item label</h3>
     <vaadin-demo-snippet id="list-box-presentation-demos-custom-label">
       <template preserve-content>
-        <b>User</b>
+        <label>User</label>
```

Comment:
I noticed you didn't add any styles for the label (though I mentioned you can do it if you like) so it won't look bold like before. I think it's fine like this too, but if we want it to just look a bit different we could add some styles for it in the `index.html` file like I suggested. It's up to you.

Reply:
I think there is no need to add style for it.